File: store/src/main/java/com/kjj/store/dto/StoreDto.java
package com.kjj.store.dto;

import com.kjj.store.entity.Store;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class StoreDto {
    private String name;
    private String info;
    private String image;

    public static StoreDto from(Store store) {
        return new StoreDto(
                store.getName(),
                store.getInfo(),
                store.getImage()
        );
    }
}

File: store/src/main/kotlin/com/kjj/store/dto/StoreDto.kt
package com.kjj.store.dto

import com.kjj.store.entity.Store

data class StoreDto(
    val name: String,
    val info: String,
    val image: String?
) {
    companion object {
        fun from(store: Store): StoreDto {
            return StoreDto(
                store.name,
                store.info,
                store.image
            )
        }
    }
}